routes/personality_route.py

```
@@ -176,116 +172,92 @@ def post(
+            "}\n"
+            "예를 들어, 만약 대화에서 내향적 성향이 강화되면 {\"ei\": \"NEW_I\", \"sn\": \"NO_CHANGE\", \"tf\": \"NO_CHANGE\", \"jp\": \"NO_CHANGE\"}와 같이 응답하세요."
+        )
+        user_prompt = f"최근 {days}일간 사용자 대화:\n{conversation_text}"
+
+        gpt_result = gpt_call(system_prompt, user_prompt)
+        print("[DEBUG] GPT 분석 결과:", gpt_result)
+
+        try:
+            changes = json.loads(gpt_result)
+        except Exception as e:
+            return HTTPException(
```

Ruff (0.8.2)

`HTTPException` may be undefined, or defined from star imports

(F405)